src/bci_build/package/basecontainers.py

```diff
@@ -355,7 +355,7 @@ def _get_minimal_kwargs(os_version: OsVersion):
         name="slm60-kernel-module-devel",
         pretty_name="SUSE Linux Micro 6.0 Kernel module development",
         logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
-        os_version=OsVersion.SLE16_0,
+        os_version=(os_version := OsVersion.SLE16_0),
         supported_until=_SUPPORTED_UNTIL_SLE.get(os_version),
```

Comment:
so this line is wrong independent of the os_version instability, as the lifecycle of the container does not depend on SLE.

Reply:
filed https://github.com/SUSE/BCI-dockerfile-generator/pull/2396 to fix this.